src/components/UI/downloads/DownloadsHero.tsx

```diff
@@ -67,14 +67,12 @@ export const DownloadsHero: FC<DownloadsHero> = ({
           </Stack>
 
           <Stack
-            py={{ base: 0 }}
```

Comment:
Padding is still required for [tablet/desktop ](https://www.figma.com/file/ekzIgwyeVKLtFSAcnA0Q0D/geth-website?node-id=1043%3A8879&t=rol3tipDhhgzwHrC-0)(`md` and higher). Seems like [padding for the content in mobile](https://www.figma.com/file/ekzIgwyeVKLtFSAcnA0Q0D/geth-website?node-id=1077%3A9400&t=rol3tipDhhgzwHrC-0) should be updated too 

<img width="1070" alt="Screen Shot 2022-11-21 at 22 13 27" src="https://user-images.githubusercontent.com/948922/203190393-4edf18df-4cd3-45f8-9eef-d0036f4a430a.png">